Pages/List/list.html

```diff
@@ -152,6 +208,12 @@ <h3>Default List w/ No Bullets</h3>
                     <li>Item3</li>
                 </ul>
             </div>
+            <p class="body-r">Copy the given code to create Default List with No Bullets</p>
+            <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%283%2C26%2C77%2C1%29&t=blackboard&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Fira+Code&fs=13px&lh=152%25&si=false&es=2x&wm=false&code=%253Cul%2520class%253D%2522list-style-none%2522%253E%250A%2509%253Cli%253EItem1%253C%252Fli%253E%250A%2509%253Cli%253EItem2%253C%252Fli%253E%250A%2509%253Cli%253EItem3%253C%252Fli%253E%250A%253C%252Ful%253E"
+  style="width: 50rem; height: 40rem; border:0; transform: scale(1); overflow:hidden;"
+  sandbox="allow-scripts allow-same-origin">
```

Comment:
I am not sure about these inline styles for iframes if they comes along with carbon. If no then may be you can plan to use a class for these properties as they are repeating.

Reply:
Whenever I import iframe from carbon it comes with this inline style, the only thing that I have changed is the unit of width and height (from px to rem)